api/src/main/java/org/pmiops/workbench/db/model/DbConceptSet.java

```diff
@@ -188,4 +203,89 @@ public void setParticipantCount(int participantCount) {
   public void setConceptIds(Set<Long> conceptIds) {
     this.conceptIds = conceptIds;
   }
+
+  public static Builder builder() {
+    return new Builder();
+  }
+
+  public static class Builder {
+    private long conceptSetId;
+    private int version;
+    private String name;
+    private short domain;
+    private Short survey;
+    private String description;
+    private long workspaceId;
+    private DbUser creator;
+    private Timestamp creationTime;
+    private Timestamp lastModifiedTime;
+    private int participantCount;
+    private Set<Long> conceptIds = new HashSet<>();
+
+    private Builder() {}
+
+    public DbConceptSet.Builder addConceptSetId(long conceptSetId) {
+      this.conceptSetId = conceptSetId;
+      return this;
+    }
+
+    public DbConceptSet.Builder addVersion(int version) {
+      this.version = version;
+      return this;
+    }
+
+    public DbConceptSet.Builder addName(String name) {
+      this.name = name;
+      return this;
+    }
+
+    public DbConceptSet.Builder addDomain(Short domain) {
+      this.domain = domain;
+      return this;
+    }
+
+    public DbConceptSet.Builder addSurvey(Short survey) {
+      this.survey = survey;
+      return this;
+    }
+
+    public DbConceptSet.Builder addDescription(String description) {
+      this.description = description;
+      return this;
+    }
+
+    public DbConceptSet.Builder addWorkspaceId(long workspaceId) {
+      this.workspaceId = workspaceId;
+      return this;
+    }
+
+    public DbConceptSet.Builder addCreator(DbUser creator) {
+      this.creator = creator;
+      return this;
+    }
+
+    public DbConceptSet.Builder addCreationTime(Timestamp creationTime) {
+      this.creationTime = creationTime;
+      return this;
+    }
+
+    public DbConceptSet.Builder addLastModifiedTime(Timestamp lastModifiedTime) {
+      this.lastModifiedTime = lastModifiedTime;
+      return this;
+    }
+
+    public DbConceptSet.Builder addParticipantCount(int participantCount) {
+      this.participantCount = participantCount;
+      return this;
+    }
+
+    public DbConceptSet.Builder addConceptIds(Set<Long> conceptIds) {
+      this.conceptIds = conceptIds;
+      return this;
+    }
+
+    public DbConceptSet build() {
+      return new DbConceptSet(this);
+    }
+  }
```

Comment:
@jaycarlton added your recommended builder pattern to help construction.

Reply:
Awesome!  You can.  also make. a. vaildation function check required args are present or throw.

nit: I'm used to see the work happen in build(), there's. no reason it can't work this way.